desktop/src/main/java/haveno/desktop/main/funds/withdrawal/WithdrawalView.java

```diff
@@ -141,20 +134,15 @@ public void initialize() {
         withdrawToTextField = addTopLabelInputTextField(gridPane, ++rowIndex,
                 Res.get("funds.withdrawal.toLabel", Res.getBaseCurrencyCode())).second;
 
-        feeToggleGroup = new ToggleGroup();
-
-        final Tuple4<Label, TextField, RadioButton, RadioButton> feeTuple3 = FormBuilder.addTopLabelTextFieldRadioButtonRadioButton(gridPane, ++rowIndex, feeToggleGroup,
+        final Tuple3<Label, TextField, Button> feeTuple3 = FormBuilder.addTopLabelTextFieldRadioButtonRadioButton(gridPane, ++rowIndex, "",
                 Res.get("funds.withdrawal.receiverAmount", Res.getBaseCurrencyCode()),
-                "",
-                Res.get("funds.withdrawal.feeExcluded"),
-                Res.get("funds.withdrawal.feeIncluded"),
+                Res.get("funds.withdrawal.sendMax", Res.getBaseCurrencyCode()),
```

Comment:
Remove `Res.getBaseCurrencyCode()` since the translation does not have {0} variable.